modules/boilerplate/security.js

```diff
@@ -28,23 +22,28 @@ const defaultSecurityDomains = [
     'cdn.jsdelivr.net'
 ];
 
+const childSrc = defaultSecurityDomains.concat(['www.google.com']);
+
 const helmetSettings = helmet({
     contentSecurityPolicy: {
         directives: {
             defaultSrc: defaultSecurityDomains,
-            frameSrc: defaultSecurityDomains.concat(['www.google.com']),
+            childSrc: childSrc,
+            frameSrc: childSrc,
             styleSrc: defaultSecurityDomains.concat(["'unsafe-inline'", 'fonts.googleapis.com']),
             connectSrc: defaultSecurityDomains.concat(['ws://127.0.0.1:35729/livereload']), // make dev-only?,
             imgSrc: defaultSecurityDomains.concat(['data:']),
-            scriptSrc: defaultSecurityDomains.concat(["'unsafe-eval'", "'unsafe-inline'"])
-        }
+            scriptSrc: defaultSecurityDomains.concat(["'unsafe-eval'", "'unsafe-inline'"]),
+            reportUri: 'https://sentry.io/api/226416/csp-report/?sentry_key=53aa5923a25c43cd9a645d9207ae5b6c'
```

Comment:
This is a public DSN rather than a private key so to my understanding is fine to be in source control like this, only the private DSN used for server-side application logic needs to come from secrets.

Reply:
👍